flink-ai-flow/ai_flow/java/client/src/main/java/com/aiflow/client/MetadataClient.java

```diff
@@ -137,11 +138,11 @@ public DatasetMeta registerDatasetWithCatalog(String name, String catalogName, S
                                                   String catalogConnectionUri, String catalogTable, String catalogDatabase) throws Exception {
         DatasetProto.Builder dataset = DatasetProto.newBuilder().setName(name).setDataFormat(stringValue(null))
                 .setDescription(stringValue(null)).setUri(stringValue(null))
-                .putAllProperties(null).setSchema(SchemaProto.newBuilder().addAllNameList(null).addAllTypeList(null))
+                .putAllProperties(Collections.emptyMap()).setSchema(SchemaProto.newBuilder().addAllNameList(Collections.emptyList()).addAllTypeList(Collections.emptyList()))
```

Comment:
It's recommented to pass the `properties` parameter for users.

Reply:
This is deserved to be discussed, but we'd better keep it consistent with python client